pages/index.js

```diff
@@ -1,59 +1,73 @@
 import Head from "next/head";
 import { Layout } from "../components/organisms/Layout";
+import { ActionButton } from "../components/atoms/ActionButton";
 
 export default function Home() {
   return (
-    <div>
+    <Layout
+      bannerTitle="Service Canada Labs"
+      bannerText="Make government work for you"
+      bannerBecomeTesterText="Become a Tester"
+      bannerStartTestingText="Start testing"
+    >
       <Head>
         <title>Alpha Site</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Layout
-        bannerTitle="Service Canada Labs"
-        bannerText="Make government work for you"
-        bannerBecomeTesterText="Become a Tester"
-        bannerStartTestingText="Start testing"
-      >
-        <section>
-          <h2>About us</h2>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus
-            consectetur urna nec tempor tristique. Vestibulum ante ipsum primis
-            in faucibus orci luctus et ultrices posuere cubilia curae; Nulla at
-            condimentum odio, quis porttitor eros. Quisque laoreet pretium
-            purus, sed maximus turpis. Quisque ut magna vel dui placerat aliquet
-            in non orci.
-          </p>
+      <div className="bg-gray-light-200 py-6">
+        <div className="layout-container flex">
+          <ActionButton
+            text="Start testing"
+            secondary
+            className={"mr-4"}
+            dataCyButton={"become-tester-button"}
+          />
+          <ActionButton
+            text="Become tester"
+            dataCyButton={"start-testing-button"}
+          />
```

Comment:
This is probably "Become a tester", although in [the newer mockups](https://www.figma.com/file/t5uDkPa86nsS3ssFESX2Em/BDM-DECD-Alpha-Site-Sprint-2?node-id=105%3A3801), the buttons are gone anyway. 💣